admin-guide/en/single-signon.md

```diff
@@ -0,0 +1,68 @@
+# Introduction 
```

Comment:
One thing to note is that a page should only have one h1 heading, which is the single # in markdown, and that should be the title of the page. Other sections should be a h2 (or ## in markdown). This is important for accessibility as the headings are used by screenreaders.